python-backend/app/api/permits/permit_amendment/models/permit_amendment.py

```diff
@@ -41,16 +43,44 @@ def json(self):
             'issue_date':
             self.issue_date.isoformat() if self.issue_date else None,
             'authorization_end_date':
-            self.authorization_end_date.isoformat() if self.authorization_end_date else None
+            self.authorization_end_date.isoformat() if self.authorization_end_date else None,
+            'related_documents': [x.json() for x in self.documents]
         }
 
+    @classmethod
+    def create(cls,
+               permit,
+               received_date,
+               issue_date,
+               authorization_end_date,
+               user_kwargs,
+               permit_amendment_type_code='AMD',
+               permit_amendment_status_code='ACT',
+               save=False):
```

Comment:
I think the other `save` values are defaulted to `True`.

Reply:
Correct, good catch. Thanks.